Sources/SwifterSwift/CoreLocation/CLLocationExtensions.swift

```diff
@@ -70,6 +69,21 @@ public extension CLLocation {
         return (degrees+360).truncatingRemainder(dividingBy: 360)
     }
 
+    /// SwifterSwift: Retrieve the first placemark of a location.
+    ///
+    /// - Parameter completion: completion handler to run after reverseGeocodeLocation finishes.
+    /// - Returns: An array of  placemarks for a given location or nil if it didn't find any.
```

Comment:
The function doesn't return, maybe it is better to describe this on the completion paramter result :)